src/flag_gems/ops/isin.py

```diff
@@ -0,0 +1,294 @@
+import torch
+import triton
+import triton.language as tl
+
+from flag_gems.utils.libentry import libentry
+
+from .unique import _unique2
+
+
+def launch_arg(BLOCK_M, BLOCK_N, N, num_warps):
+    return BLOCK_M, min(BLOCK_N, triton.next_power_of_2(N)), num_warps
+
+
+@triton.jit
+def all_fn(a, b):
+    return a and b
+
+
+@triton.jit
+def any_fn(a, b):
+    return a or b
+
+
+@triton.jit
+def isin_by_comparation_impl(
+    global_pid,
+    in0_ravel_ptr: tl.tensor,
+    in1_ravel_ptr: tl.tensor,  # in
+    out_ptr: tl.tensor,  # out
+    M: int,  # num_tasks
+    N: int,  # num_tasks_1
+    BLOCK_M: tl.constexpr,  # tile_size
+    BLOCK_N: tl.constexpr,  # tile_size_1
+    invert: tl.constexpr,
+):
+    row_off = global_pid * BLOCK_M
+    rows = row_off + tl.arange(0, BLOCK_M)[:, None]
```

Comment:
Can we use slice like `[:，none]` in a triton.jit kernel now?

Reply:
> Can we use slice like `[:，none]` in a triton.jit kernel now?

No, we can only use this to broadcast, but not slice.